app/admin/events.rb

```diff
@@ -79,4 +86,15 @@
     end
     active_admin_comments
   end
+
+  sidebar 'Featured Projects', only: :show do
+    table_for event.featured_projects do
+      column :project
+      column 'Actions' do |fp|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.